cmd/README.md

```diff
@@ -188,8 +188,32 @@ where `test/test.js` is the path of the file.
 You have the flexibility to write tests in quite a variety of ways, open to you to explore.
 
 ## Running your application
-For the purpose of *testing* APIs or prototyping user interfaces, you can run your DNA using `hc run`.
-This will start the application and open a WebSocket on port `8888` or any other port specified by the `-p`/`--port` option. The `-b`/`--package` flag will build your DNA before running it. By default, none of the data your application is putting on the source chain gets persisted. To persist data onto the file system, use the `--persist` flag.
+For the purpose of *testing* APIs or prototyping user interfaces, you can run the DNA of a hApp, from the directory it's contained. The most basic way to do this is to run:
```

Comment:
This is an opportunity for us to clean up some of these concepts. What's the natural way to word this? The problem with how it's written is that there is "*the* DNA" for a hApp, where a hApp typically consists of many DNAs.

Is "run *a* DNA" how we want to say it? It's weird because DNA in biology is just "stuff", not a "thing". "run DNA" sounds more natural, but it's talking about stuff, not a thing. Are we going to (have we already) co-opted "DNA" to refer to a thing?

```suggestion
For the purpose of *testing* APIs or prototyping user interfaces, you can run a DNA from the directory it's contained. The most basic way to do this is to run:
```

Reply:
good question. Not sure about the bigger picture, but I like this suggestion, merged it